application/proxy/tariff/tariff.go

```diff
@@ -0,0 +1,243 @@
+//
+// Copyright 2019 Insolar Technologies GmbH
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package tariff
+
+import (
+	"github.com/insolar/insolar/insolar"
+	"github.com/insolar/insolar/logicrunner/common"
+	"github.com/insolar/insolar/logicrunner/goplugin/foundation"
+)
+
+// PrototypeReference to prototype of this contract
+// error checking hides in generator
+var PrototypeReference, _ = insolar.NewReferenceFromBase58("1111WGU1izeiSyT2XMdDQodp3B3BwmCpYDGZcqQbwd.11111111111111111111111111111111")
```

Comment:
use something like MustNewReferenceFromBase58 here, not just ignore error

Reply:
proxy